### PR TITLE
Added my video course

### PR DESCRIPTION
**Pull request by Dawny33**
- Added the video course by Packt. 

- I am the author [produced from my published work (https://www.packtpub.com/application-development/julia-cookbook/)]

- Released on Jan 31st, 2017

### CODE REVIEW

**Comment by ViralBShah** [comment]
Perhaps this is more appropriate to add under the books section. Maybe we need a clear distinction between free content, and professional/to be paid for content.

**Comment by Dawny33** [comment]
@ViralBShah Makes sense. Added to the `Books` section. Pl review.

>Maybe we need a clear distinction between free content, and professional/to be paid for content.
As all the books are `paid` resources, I guess the md is already sorted :)

File: learning/index.md [hunk 1]
```diff
@@ -57,6 +58,7 @@ title:  Learning Julia
 
 - [Julia Tutorial](https://www.youtube.com/watch?v=AyvyVS6u8AM) by [Chase Coleman](http://www.chasegcoleman.com/) at PyDataDC 2016. Supporting material [here](https://github.com/cc7768/PyDataDC_julia).
 
+
```
Comment (ararslan) [inline learning/index.md]: Unrelated change

Comment (Dawny33) [inline learning/index.md]: @ararslan Fixed. Pl review

File: learning/index.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ title:  Learning Julia
     Explore the world of data science from scratch with Julia by your side
 - [Julia for Data Science](https://technicspub.com/analytics/) by Zacharias Voulgaris PhD (415 pages; published: 2016-09-01; ISBN: 9781634621304). Master the essentials of data science through the Julia programming ecosystem (no prior knowledge of the language is required), accompanied by a variety of interesting examples and exercises.
 - [Julia Cookbook](https://www.packtpub.com/application-development/julia-cookbook/) by Jalem Raj Rohit - published by Packt Publishing (172 pages; published: 2016-09; ISBN: 9781785882012)
+- [Julia Solutions](https://www.packtpub.com/big-data-and-business-intelligence/julia-solutions-video) by Jalem Raj Rohit - A comprehensive guide to learn data science for a Julia programmer - Produced by Packt Publishing (2 hours and 52 minutes long; published: January 31, 2017; ISBN: 9781787283299)
```
Comment (ararslan) [inline learning/index.md]: Should be "learning" rather than "learn"

Comment (Dawny33) [inline learning/index.md]: Hi @ararslan. I took the line from the course's page: (https://www.packtpub.com/big-data-and-business-intelligence/julia-solutions-video).

`to learn` and `for learning` are both the same IMHO :)

Comment (ararslan) [inline learning/index.md]: Ah, I was parsing it differently. What you have now is fine.
